code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -25,8 +25,7 @@
 	var/pressure_direction = 0
 
 	var/datum/excited_group/excited_group
-	var/excited = 0
-	var/recently_active = 0
```

Comment:
this var proved to be redundant and also maybe just actively harming performance??